tools/eval/run_register.py

```
@@ -300,28 +443,52 @@ def print_delta(baseline_stats, variant_stats, n_baseline, n_variant):
         print(f"  {metric:<26} {change:>+8.2f} {sigma:>+8.2f} {degrees:>6.1f} "
               f"{threshold if threshold else 0:>6.3f}   {reading} [{role}]")
+        if metric == PRIMARY_METRIC:
+            decision = {
+                "metric": metric,
+                "delta": change,
+                "sigma": sigma,
+                "degrees_of_freedom": degrees,
+                "threshold_95": threshold,
+                "reading": reading,
+            }
     print(f"\n  Read the [decides] row alone as the result. The [describes] rows are "
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)